tests/unit/source_code/test_path_lookup_simulation.py

```diff
@@ -33,9 +33,9 @@
             ],
             3,
         ),
-        (["simulate-sys-path", "via-sys-path", "run_notebook_1.py"], 1),
-        (["simulate-sys-path", "via-sys-path", "run_notebook_2.py"], 1),
-        (["simulate-sys-path", "via-sys-path", "run_notebook_4.py"], 2),
+        (["simulate-sys-path", "via-sys-path", "run_notebook_1.py"], 2),
```

Comment:
These tests required updating as the known dependencies are now registered where before they were not